pkg/config/config.go

```diff
@@ -39,6 +39,8 @@ type Config struct {
 	HostsDir         []string `toml:"hosts_dir"`
 	Experimental     bool     `toml:"experimental"`
 	HostGatewayIP    string   `toml:"host_gateway_ip"`
+	// CDISpecDirs is a list of directories in which CDI specifications can be found.
+	CDISpecDirs []string `toml:"cdi_spec_dirs,omitempty"`
```

Comment:
bike shedding: "Dirs" vs "Path"

Reply:
"CDIPath" might be more consistent with the existing "CNIPath"?

Reply:
The point is that these are specifically directories and not files. Furthermore, the naming is consistent with this setting in other projects: containerd, cri-o, docker, singularity.

I'm happy to change if you feel strongly about it.

Reply:
> the naming is consistent with this setting in other projects: containerd, cri-o, docker, singularity.

👍